app/blueprints/template/templates/template_details.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "templates" %}
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {{ govukBackLink({
+              "text": "Back",
+              "href": url_for("template_bp.view_templates")
+            }) }}
+            <h2 class="govuk-heading-l">{{ form.template_name }}
+                <p class="govuk-body">
+                    <a class='govuk-link govuk-link--no-visited-state'
+                       target="_blank"
+                       href='{{ url_for("index_bp.preview_form", form_id=form.form_id) }}'>
+                        Preview template (opens in a new tab)
+                    </a>
+                </p>
+            </h2>
```

Comment:
The link should not be inside the h2

```suggestion
            <h2 class="govuk-heading-l">{{ form.template_name }}</h2>
            <p class="govuk-body">
                <a class='govuk-link govuk-link--no-visited-state'
                   target="_blank"
                   href='{{ url_for("index_bp.preview_form", form_id=form.form_id) }}'>
                    Preview template (opens in a new tab)
                </a>
            </p>
```

Reply:
Fixed